shared/src/models/personnel.ts

```diff
@@ -39,12 +43,31 @@ export class Personnel {
     @Type(() => CanCaterFor)
     public readonly canCaterFor: CanCaterFor;
 
+    /**
+     * Patients in this range are preferred over patients that are more far away (even if they are less injured).
+     * Guaranteed to be <= {@link maxTreatmentRange}.
+     */
+    @IsNumber()
+    @Min(0)
+    @Max(maxTreatmentRange)
+    public readonly overrideTreatmentRange: number;
+
+    /**
+     * Only patients in this range around the personnel's position can be treated.
+     * Guaranteed to be <= {@link maxTreatmentRange}.
+     */
+    @IsNumber()
+    @Min(0)
+    @Max(maxTreatmentRange)
+    public readonly treatmentRange: number;
+
     @ValidateNested()
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
     /**
-     * if undefined, is in vehicle with {@link vehicleId} or in transfer
+     * If undefined, the personnel is either in the vehicle with {@link this.vehicleId} or in transfer.
+     * Exclusive-or to {@link transfer}
```

Comment:
```suggestion
```

If the personnel is in a vehicle, neither `transfer` nor `position` has a value. Therefore this is not an exclusive or.

Additionally, I would change the comment on transfer to 
```
     * If undefined, the personnel is either in the vehicle with {@link this.vehicleId} or has a {@link position}.
```

Reply:
@Dassderdie done

Reply:
Marking this as resolved, @Dassderdie if you think otherwise undo or ping me in this conversation.